utils/hare-bootstrap

```diff
@@ -199,6 +199,19 @@ is_localhost() {
     [[ -e $path && $(cat $path) == $node ]] && return 0 || return 1
 }
 
+check_consul_failure() {
+
+    while read node bind_ip; do
+        if ssh -n $node "sudo systemctl --quiet --state=failed is-failed hare-consul-agent"; then
```

Comment:
Can we use `ssh_error_info` instead?

Reply:
@knekrasov ssh_error_info throws error if command returns false.
In our case, command will return false in happy path. So we cant use ssh_error_info.
Please correct me if I am wrong

Reply:
Hm, good point. Ok, please ignore then.